webrender/src/platform/macos/font.rs

```diff
@@ -69,23 +70,47 @@ fn supports_subpixel_aa() -> bool {
     data[0] != data[1] || data[1] != data[2]
 }
 
-fn get_glyph_metrics(ct_font: &CTFont, glyph: CGGlyph) -> GlyphMetrics {
+fn get_glyph_metrics(ct_font: &CTFont,
+                     glyph: CGGlyph,
+                     x_subpixel: SubpixelOffset,
```

Comment:
I'll do that after https://github.com/servo/webrender/pull/798#issuecomment-275822832 lands. I got a couple of things to rewrite but I'd like to just land some PRs first.